src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -117,7 +117,7 @@ public async Task<(Error, GitHubUser)> GetUserByLogin(string login)
             }
             catch (Exception ex)
             {
-                return (Errors.GitHubApiFailed(_url, ex.Message), null);
+                return (Errors.GitHubApiFailed(_url, ex.InnerException?.Message ?? ex.Message), null);
```

Comment:
can you write these detailed diagnostic info using Log.Write?, the Error message is mainly for writers, not for troubleshooting

Reply:
so what the message do you want to return to the end user?

Reply:
```suggestion
                Log.Write(ex);
                return (Errors.GitHubApiFailed(_url, ex.Message), null);
```

Reply:
if we already have the really message, why not directly return to end user? let's say we are using by some developers, they can find more info from here. And yes, we can also write to disgnostic.

Reply:
sure